userspace/libscap/engine/modern_bpf/scap_modern_bpf.c

```diff
@@ -207,7 +240,7 @@ int32_t scap_modern_bpf__init(scap_t* handle, scap_open_args* oargs)
 	 * Validation of `cpus_for_each_buffer` is made inside libpman
 	 * since this is the unique place where we have the number of CPUs
 	 */
-	if(pman_init_state(params->verbose, params->buffer_bytes_dim, params->cpus_for_each_buffer, params->allocate_online_only))
+	if(pman_init_state(libbpf_verbosity, params->buffer_bytes_dim, params->cpus_for_each_buffer, params->allocate_online_only))
```

Comment:
here we have something wrong, `libbpf_verbosity` was removed some commits ago, now we have `params->verbose`